electron/main.js

```diff
@@ -192,15 +192,23 @@ function showMainWindow() {
     const {hostname = '', certificate = {}, error} = request;
 
     if (typeof error !== 'undefined') {
+      console.error(error);
       return cb(-2);
     }
 
-    if (certutils.hostnameShouldBePinned(hostname) && !(certutils.verifyPinning(hostname, certificate))) {
-      console.error(`Certutils could not verify certificate pinning for ${hostname}.`);
-      cb(-2);
-    } else {
-      cb(-3);
+    if (certutils.hostnameShouldBePinned(hostname)) {
+      const pinningResult = certutils.verifyPinning(hostname, certificate);
+      if (pinningResult.verifiedFingerprints === false) {
```

Comment:
Right now I'm setting `verifiedFingerprints` to `undefined` if no checkable fingerprints are found. Your requested change would mean that this line always fails, if no checkable fingerprints are found - this is not our desired result.

I specifically chose `undefined` because it defines an unknown state - because when no fingerprints where found for checking, `verifyPinning()` cannot tell if it can verify or not.